docs/queue.md

```diff
@@ -146,6 +146,8 @@ functions:
             QUEUE_URL: ${construct:my-queue.queueUrl}
 ```
 
+Automatic permissions can be disabled: [read more about IAM permissions](permissions.md).
```

Comment:
```suggestion
Automatic permissions can be disabled: [read more about IAM permissions](permissions.md#disabling-automatic-permissions).
```

Reply:
The link is intentional ("read more about IAM permissions"): the separate page recaps how permissions work in Lift in general.

I'd rather have users read the whole page to understand how it works. At the moment the documentation on IAM permissions is a bit vague (i.e. "it works magically"), so I think that separate page should do 2 jobs:

- explain how permissions work (to clear up confusion)
- show how to disable

Reply:
I get your point, it makes sense ! :+1: